[RN-v2.4.0] useAccessibilityLabel config parameter (#24013)
DataDog/documentation: content/en/real_user_monitoring/mobile_and_tv_monitoring/advanced_configuration/reactnative.md

**Type**: ProxyConfiguration<br/>
Optional [proxy configuration][13].

`useAccessibilityLabel`
: Optional<br/>
**Type**: Boolean<br/>
**Default**: `true`<br/>

> Determines whether the accessibility labels can be used to name RUM actions (default is true).

Reviewer: ```suggestion
Determines whether the accessibility labels are used to name RUM actions (default is true).
```
I think?
